xmake/rules/platform/linux/bpf/xmake.lua

```diff
@@ -46,6 +46,7 @@ rule("platform.linux.bpf")
             targetarch = "__TARGET_ARCH_powerpc"
         end
         target:add("includedirs", path.directory(headerfile))
+        target:set("optimize", "faster")
         batchcmds:show_progress(opt.progress, "${color.build.object}compiling.bpf %s", sourcefile)
         batchcmds:mkdir(path.directory(objectfile))
         batchcmds:compile(sourcefile, objectfile, {configs = {force = {cxflags = {"-target bpf", "-g"}}, defines = targetarch}})
```

Comment:
Why not add `-O2` to here? it only for compiling bpf code `cxflags = {"-target bpf", "-g", "-O2"}`

Reply:
If you prefer that way I can do it that way :)